react/components/ProductSummaryPrice.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { path } from 'ramda'
+import productSummary from '../productSummary.css'
+import { Spinner } from 'vtex.styleguide'
+
+export default class ProductSummaryPrice extends Component {
```

Comment:
Prefer functional components over class-based ones, since it doesn't have state.

(Or even [with state](https://reactjs.org/docs/hooks-intro.html))